src/main/java/com/develhope/spring/orders/service/OrderService.java

```diff
@@ -0,0 +1,77 @@
+package com.develhope.spring.orders.service;
+
+import com.develhope.spring.orders.DTOs.CreateOrderRequest;
+import com.develhope.spring.orders.DTOs.OrderResponse;
+import com.develhope.spring.orders.entity.Order;
+import com.develhope.spring.orders.entity.OrderStatus;
+import com.develhope.spring.orders.model.OrderModel;
+import com.develhope.spring.orders.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class OrderService {
+    @Autowired
+    private OrderRepository orderRepository;
+
+    public OrderService(OrderRepository orderRepository) {
+        this.orderRepository = orderRepository;
+
+    }
+
+    public List<OrderResponse> getAllOrders() throws Exception {
+        List<Order> orders = orderRepository.findAll();
+        if (orders.isEmpty()) {
+            throw new Exception("ops, looks like there is nothing here.");
```

Comment:
In questa Exception aggiungerei un errore più esplicito rispetto alla lista di ordini vuota.